pkg/controller/statusmanager/pod_status.go

```diff
@@ -86,7 +86,11 @@ func (status *StatusManager) SetFromPods() {
 			progressing = append(progressing, fmt.Sprintf("DaemonSet %q update is rolling out (%d out of %d updated)", dsName.String(), ds.Status.UpdatedNumberScheduled, ds.Status.DesiredNumberScheduled))
 			dsProgressing = true
 		} else if ds.Status.NumberUnavailable > 0 {
-			progressing = append(progressing, fmt.Sprintf("DaemonSet %q is not available (awaiting %d nodes)", dsName.String(), ds.Status.NumberUnavailable))
+			if strings.Contains(dsName.String(), "multus-admission-controller") {
+				progressing = append(progressing, fmt.Sprintf("DaemonSet %q is waiting for other operators to become ready (awaiting %d nodes)", dsName.String(), ds.Status.NumberUnavailable))
```

Comment:
it doesn't make sense to include the `NumberUnavailable` here because we only want the special case message if it's unavailable on all nodes. So I guess the previous condition should be `isNonCritical(ds) && ds.Status.NumberAvailable == 0` and then don't print a number here